users/templates/base.html

```diff
@@ -17,8 +17,8 @@
     />
     <link
         rel="shortcut icon"
-        href="{% static 'images/favicon.ico' %}"
-        integrity="{% staticinline 'images/favicon.ico' encode="sri" %}"
+        href="{% static 'users/images/favicon.ico' %}"
+        integrity="{% staticinline 'users/images/favicon.ico' encode="sri" %}"
```

Comment:
Why do you move `favicon.ico` to `users` app? This is same for all apps in Sheetstorm and it should stay in main directory

Reply:
Alright, I thought it was a bug, since images everywhere are placed under `app/static/app` and **favicon.ico** is located in the **users** app directory either way. It is also called only once in entire app in *base.html*...
Which is also a part of the **users** app... 😕 
I'd say in that case it should go to somewhere like `sheetstorm/static` along with base.html, but that directory is ignored by git.
It is very confusing to have both directories `users/static/images` and `users/static/users/images` I don't think I'm the only one who would consider this a bug after stumbling upon it.